README.md

```diff
@@ -114,7 +114,7 @@ use the `/dev/tty.usbmodemXXX` device node involving how it operates under the h
 When developing this project, the following tools are required:
 
 * [git](https://git-scm.com)
-* [meson](https://mesonbuild.com/Getting-meson.html) (version 0.61 or greater)
+* [meson](https://mesonbuild.com/Getting-meson.html) (version 1.5.1 or greater)
```

Comment:
We've had it confirmed by another contributor that v1.1.1 also works. The changelog suggests that it was fixed in 0.63. Given this, let's bump the Meson requirement here to just 0.63 and see if anyone else complains with at least that. Think saying 1.5.1 as it's the latest release is a little too strict.

Reply:
Note that bumping the meson version requirement should be done in the main `meson.build` first, there it'll produce a warning even for those who won't look at the readme